app/ts/components/CopyButton.tsx

```diff
@@ -1,14 +1,19 @@
 import { useSignal, useSignalEffect } from '@preact/signals'
-import { ComponentChild } from 'preact'
+import { ComponentChild, JSX } from 'preact'
+import { useNotification } from '../context/Notification.js'
 import * as Icon from './Icon/index.js'
 
 type CopyButtonProps = {
 	value: string
-	label: ComponentChild
+	withLabel?: boolean
+	// for deprecation: use `withLabel` to display text label
```

Comment:
add issue about this

Reply:
created issue here: https://github.com/DarkFlorist/lunaria/issues/221